src/services/analytics/gtm.ts

```diff
@@ -103,8 +104,17 @@ export const gtmTrack = (eventData: AnalyticsEvent): void => {
     eventAction: eventData.action,
   }
 
+  if (eventData.event === EventType.META) {
```

Comment:
We could also track the eventType in general not just if its a meta event

Reply:
I adjusted this to be more generalized and track the event type in all cases. Don't see a strong reason to only do it for metadata.